src/main/resources/ServerRoutes.yaml

```diff
@@ -43,4 +43,89 @@ methodhandlers:
 path: /echo_body
 methodhandlers:
   - method: POST
-    handler: POST_REQUEST_HANDLER
\ No newline at end of file
+    handler: POST_REQUEST_HANDLER
+---
+path: /style.css
+methodhandlers:
+  - method: GET
+    handler: FILE_REQUEST_HANDLER
+---
+path: /dist/bundle.js
+methodhandlers:
+  - method: GET
+    handler: FILE_REQUEST_HANDLER
+---
+path: /images/apple.svg
+methodhandlers:
+  - method: GET
+    handler: FILE_REQUEST_HANDLER
+---
+path: /images/bacon.svg
+methodhandlers:
+  - method: GET
+    handler: FILE_REQUEST_HANDLER
+---
+path: /images/broccoli.svg
+methodhandlers:
+  - method: GET
+    handler: FILE_REQUEST_HANDLER
+---
+path: /images/cherry.svg
+methodhandlers:
+  - method: GET
+    handler: FILE_REQUEST_HANDLER
+---
+path: /images/cupcake.svg
+methodhandlers:
+  - method: GET
+    handler: FILE_REQUEST_HANDLER
+---
+path: /images/cookie.svg
+methodhandlers:
+  - method: GET
+    handler: FILE_REQUEST_HANDLER
+---
+path: /images/donut.svg
+methodhandlers:
+  - method: GET
+    handler: FILE_REQUEST_HANDLER
+---
+path: /images/egg.svg
+methodhandlers:
+  - method: GET
+    handler: FILE_REQUEST_HANDLER
+---
+path: /images/fries.svg
+methodhandlers:
+  - method: GET
+    handler: FILE_REQUEST_HANDLER
+---
+path: /images/ice-cream.svg
+methodhandlers:
+  - method: GET
+    handler: FILE_REQUEST_HANDLER
+---
+path: /images/pepper.svg
+methodhandlers:
+  - method: GET
+    handler: FILE_REQUEST_HANDLER
+---
+path: /images/pizza.svg
+methodhandlers:
+  - method: GET
+    handler: FILE_REQUEST_HANDLER
+---
+path: /images/soda.svg
+methodhandlers:
+  - method: GET
+    handler: FILE_REQUEST_HANDLER
+---
+path: /images/strawberry.svg
+methodhandlers:
+  - method: GET
+    handler: FILE_REQUEST_HANDLER
+---
+path: /images/taco.svg
+methodhandlers:
+  - method: GET
+    handler: FILE_REQUEST_HANDLER
```

Comment:
Having to add a route for every single file seems hard to manage, having some sort of "public" directory and putting assets into that might alleviate having to reopen the router for every new image, style, or script.

Reply:
refactored in https://github.com/olgaboiar/MintHTTPServer/pull/14/commits/0d83aab23459d64f950adcc1a04fce85f34df599